Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,101 @@
+| Fire | Medium | permanent Disk and Node loss in the affected zone |
+| Flood | Low | permanent Disk and Node loss in the affected zone |
+| Earthquake | Very Low | permanent Disk and Node loss in the affected zone |
+| Storm/Tornado | Low | permanent Disk and Node loss in the affected fire zone |
+| Cyber threat | High | permanent loss or compromise of data on affected Disk and Node |
+| Software Bug | High | permanent loss or compromise of data that trigger the bug up to data on the whole physical machine |
+
+These failure cases can result in temporary (T) or permanent (P) loss of the resource or data within.
+Additionally there are a lot of resources in IaaS alone that are more or less affected by these Failure Cases.
+The following table shows the impact when no redundancy or failure safety measure is in place:
+
+| Resource | Disk Loss | Node Loss | Rack Loss | Power Loss | natural catastrophy | Cyber threat | Software Bug |
+|----|----|----|----|----|----|----|----|
+| Image | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| Volume | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| User Data on RAM /CPU | | P | P | P | P | T/P | P |
+| volume-based VM | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| ephemeral-based VM | P (if on disk) | P | P | T | P (T if lucky) | T/P | P |
+| Ironic-based VM | P (all data on disk) | P | P | T | P (T if lucky) | T/P | P |
+| Secret | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| network configuration (DB objects) | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | P |
+| network connectivity (materialization) | | T (if on node) | T/P | T | P (T if lucky) | T/P | T |
+| floating IP | P (if on disk) | T (if on node) | T/P | T | P (T if lucky) | T/P | T |
+
+For some cases, this only results in temporary unavailabilities and cloud infrastructures usually have certain mechanisms in place to avoid data loss, like redundancy in storage backends and databases.
+So some of these outages are easier to mitigate than others.
+A possible way to classify the failure cases into levels considering the matrix of impact would be, to classify the failure cases from small to big ones.
+The following table shows such a classification, the occurance probability of a failure case of each class and what resources with user data might be affected.
+
+:::caution
+
+This table only contains examples of failure cases and examples of affected resources.
+This should not be used as a replacement for a risk analysis.
+The column **user hints** only show examples of standards that may provide this class of failure safety for a certain resource.
+Customers should always check, what they can do to protect their data and not rely solely on the CSP.
+
+:::
+
+| Level/Class | Probability | Failure Causes | loss in IaaS | User Hints |
```

Comment:
I am still thinking about the "user hints" column. Putting it next to the other columns is good from some perspectives, as it can be read: I want to achieve 2. Level of failuresafeness, which can be triggered by these failure causes that will result in these losses on IaaS level, so I can do, what is shown in the user hints.
But we wanted the classification not for examples for users, but mainly as a definiton for standards, so maybe we should not reference those standards here.
We could rather use an extra table with example actions(standards, "user has to to things",..) for each level/class or maybe this should rather not be in a decision record, but rather in a guide or so.

Reply:
I agree with you, @josephineSei. Linking SCS standards in this table may cause a huge synchronization effort. We always have to update this DR, if referenced standards change. I appreciate the column user hints, but would limit to a textual explanation.  See my suggestions below...